exercises/K-functions-parameters/exercise3.js

```diff
@@ -1,5 +1,9 @@
 // Write your function here
-
-var greeting = createGreeting("Daniel");
+function createGreeting (name){
+    let myGreeting = "Hello, my name is ";
+    return createGreeting + name;
+}
+let greeting = createGreeting("Daniel.");
```

Comment:
Can you make sure that this line is not changed (i.e. please keep it as ``createGreeting("Daniel")`` without the period after Daniel). How do you make sure that the test passes? What needs to be changed?